src/vs/workbench/services/assignment/common/assignmentService.ts

```diff
@@ -67,7 +67,9 @@ class WorkbenchAssignmentServiceTelemetry implements IExperimentationTelemetry {
 
 		/* __GDPR__
 			"query-expfeature" : {
-				"ABExp.queriedFeature": { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
+				"owner": "sbatten",
+				"comment": "Logs queries to the experiment service by feature for metric calculations",
```

Comment:
Can we convert this to a typescript annotation?

Reply:
this event comes from the tas module so its not 1:1